playwright-e2e/utils/api-helper.ts

```diff
@@ -15,7 +15,7 @@ export class Apihelp {
           'Content-Type': 'application/x-www-form-urlencoded',
         },
       };
-      let url = `${urlConfig.idamUrl}loginUser?username=${user.email}&password=${user.password}`;
+      let url = `https://idam-api.aat.platform.hmcts.net/loginUser?username=${user.email}&password=${user.password}`;
```

Comment:
is there any reason this was changed from the config value, as would break tests against demo?

Reply:
fixed